pyproject.toml

```
@@ -59,6 +59,8 @@ pytest-xdist = ">=2.5.0"
 types-backports = ">=0.1.3"
 types-pyyaml = ">=6.0.4"
 types-psutil = "*"
+types-pygments = "^2.17.0.20240310"
+types-colorama = "^0.4.15.20240311"
```

I'd suggest avoiding upper bounds (and shortening to just major.minor, but that's just my own taste):

```suggestion
types-pygments = ">=2.17"
types-colorama = ">=0.4"
```